src/app/modules/group/components/group-manager-list/group-manager-list.component.html

```diff
@@ -117,7 +117,7 @@
             </span>
```

Comment:
Functional detail:
there should a question mark:
> Are you sure to remove yourself from the managers of this group ~.~ **?**

![image](https://user-images.githubusercontent.com/16257159/137161647-040ad03a-8d92-47fe-98f9-d9eadb2f412e.png)

Reply:
Fixed